src/Backends/Helmholtz/MixtureParameters.cpp

```diff
@@ -287,11 +287,15 @@ static MixtureBinaryPairLibrary mixturebinarypairlibrary;
 // A fixed parameter library containing the default values
 static MixtureBinaryPairLibrary mixturebinarypairlibrary_default;
 
+namespace MixParameters {
```

Comment:
This namespace will break existing code. Please find a way to avoid it.

Reply:
Removed change to namespace and all other changes to the code. After learning more about how SWIG works, I realised the answer was to simply add the missing method into the CoolProp.i SWIG interface file and it now gets wrapped. SWIG won't recursively wrap methods in the header files, and thus it doesn't see the `apply_simple_mixing_rule` method unless it is explicitly defined.